### PR TITLE
Add a few tiny improvements to the delegate generator PR

### PR DESCRIPTION
**Pull request by zvirja**
Added a few minor improvements to the PR changes. Most of them are related to the code styling. Let me know if you disagree with some of them.

Would be really awesome if you could distribute changes across the 3 commits you already have (so in history it looks like code has been always in the final version), as this follow up doesn't bring any value per se. However, if you don't have time for that - simply merge this single commit to your branch 😉

### CODE REVIEW

File: Src/AutoFakeItEasyUnitTest/AutoFakeItEasyCustomizationTest.cs [hunk 1]
```diff
@@ -55,29 +63,27 @@ public void InitializeWithGenerateDelegatesOptionThrows()
         }
 #endif
 
-        [Fact]
+        [Fact, Obsolete]
```
Comment (blairconrad) [inline Src/AutoFakeItEasyUnitTest/AutoFakeItEasyCustomizationTest.cs]: Ah. Nice! I was not aware of this attribute.

File: Src/AutoFakeItEasyUnitTest/FixtureIntegrationTest.cs [hunk 2]
```diff
@@ -46,53 +46,51 @@ public void FixtureCanPassValuesToAbstractGenericTypeWithNonDefaultConstructor()
         [Fact]
         public void FixtureCanCreateFakeOfDelegate()
         {
-            // Fixture setup
+            // Arrange
```
Comment (blairconrad) [inline Src/AutoFakeItEasyUnitTest/FixtureIntegrationTest.cs]: How embarrassing. It seems I'd copied the "Fixture setup" etc. from the local copy of the repo I'd had before you made  a2e35d801d52f960a7479d020fdf82835c88c731.

File: Src/AutoFixture/Kernel/DelegateSpecification.cs [hunk 3]
```diff
@@ -14,17 +13,14 @@ public class DelegateSpecification : IRequestSpecification
         /// </summary>
         /// <param name="request">The specimen request.</param>
         /// <returns>
-        /// <see langword="true"/> if <paramref name="request"/> is a subclass of <see cref="MulticastDelegate"/>;
-        /// otherwise, <see langword="false"/>.
+        /// <see langword="true"/> if <paramref name="request"/> is of a delegate type, otherwise <see langword="false"/>.
```
Comment (blairconrad) [inline Src/AutoFixture/Kernel/DelegateSpecification.cs]: Better!

**Review by blairconrad — APPROVED** [review]

**Comment by blairconrad** [comment]
Thanks for the changes, @zvirja. Sorry I deviated so much from your established style. I assume at least some of the violations were for line length; have you considered adding a setting to the `.editorconfig`? (probably, but rejected for some reason)).

> Would be really awesome if you could distribute changes across the 3 commits you already have (so in history it looks like code has been always in the final version), as this follow up doesn't bring any value per se. However, if you don't have time for that - simply merge this single commit to your branch

I have time, and don't mind the work, but I actually think my 3 commits should be squashed anyhow. I would've said that they don't provide any value, as commits 2 and 3 are mostly undoing work that I did in the first one. So if you've no objections, I'll squash all 4 commits into one and update my PR.

However, if you do like the 3 separate commits I had, I'll distribute your changes.

**Comment by blairconrad** [comment]
Bah! I really should've read all my mail before I commented. I see [you like the 3 commits](https://github.com/AutoFixture/AutoFixture/pull/986#issuecomment-371776757). Will sprinkle yours in!

File: Src/AutoFakeItEasyUnitTest/AutoFakeItEasyCustomizationTest.cs (hunk 1 above)
Comment (zvirja) [reply Src/AutoFakeItEasyUnitTest/AutoFakeItEasyCustomizationTest.cs]: This is a regular .NET `Obsolete` attribute. However, the trick is that you can call obsoleted code from the other obsoleted code without warnings, so it looks finer 😄

File: Src/AutoFakeItEasyUnitTest/FixtureIntegrationTest.cs (hunk 2 above)
Comment (zvirja) [reply Src/AutoFakeItEasyUnitTest/FixtureIntegrationTest.cs]: No worries 😉 Actually, I thought that it was a matter of habit, as the previous 4 phases ugly style has been here for a while 😄

**Comment by zvirja** [comment]
> have you considered adding a setting to the .editorconfig? (probably, but rejected for some reason)).

I don't know about that possibility 😕 If it's possible to set max width in `.editorconfig` please fire an issue or PR with a hint😉

**Comment by blairconrad** [comment]
Will do. Well, I'll investigate locally first. During early days of VS 2017, some .editorconfig powers were not supported. I'll see what happens when I try it.

**Comment by blairconrad** [comment]
Bah. My mistake. `max_line_length` [is not widely supported](https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#max_line_length). In particular, VS2017 does not support it. Sorry to have gotten your hopes up.

**Comment by blairconrad** [comment]
Incorporated into AutoFixture/AutoFixture#986. Thanks, @zvirja.

**Pull request closed without merging**